### PR TITLE
Update TCIPMod.cs

### PR DESCRIPTION
**Pull request by ScalarVector1**
Made the mod do a really cool thing on load!

### CODE REVIEW

File: TCIPMod.cs [hunk 1]
```diff
@@ -4,5 +4,27 @@ namespace TCIPMod
...
+            var test2 = typeof(AmazingMod).GetMethod("MagicPatch2", BindingFlags.Instance | BindingFlags.NonPublic);
+
+            d = new Hook(test, test2, this);
+
+            d.Apply();
+        }
+
+        private object MagicPatch2(Func<Type, object> orig, Type type)
+        {
+            return new Trollface();
```
Comment (Salmacis32) [inline TCIPMod.cs:23]: :peepoWeird:

**Comment by RighteousRyan1** [comment]
Closed because its unbased

**Comment by ScalarVector1** [comment]
sad.

File: TCIPMod.cs (hunk 1 above)
Comment (steviegt6) [reply TCIPMod.cs:23]: That's not a review, you nitwit.

**Pull request closed without merging**